app/models/token.rb

```diff
@@ -0,0 +1,3 @@
+class Token < ApplicationRecord
```

Comment:
Can you add an `identifier_name` method here so that we're not just looking at UUIDs everywhere? It should probably include the `identifier_name` of the `tokenable` thing & maybe like the first few characters of the value .